src/pages/404.tsx

```diff
@@ -0,0 +1,14 @@
+import type { NextPage } from 'next'
+import { ErrorPage } from '@components/navigation/ErrorPage/ErrorPage'
+
+const NotFoundErrorPage: NextPage = () => {
+  return (
+    <ErrorPage
+      errorCode="404"
+      title="Not Found"
+      message="Page you are trying to open does not exist. You may have mistyped the address, or the page has been moved to another URL."
```

Comment:
Resource, not page. Next handles 404 on pages on it's own.

Reply:
This error page overrides default nextjs error page

Reply:
https://nextjs.org/docs/advanced-features/custom-error-page#404-page